src/components/cms/SkillFeedbackCard/SkillFeedbackCard.js

```diff
@@ -242,45 +345,72 @@ class SkillFeedbackCard extends Component {
 
     return (
       <Paper>
-        <Title>Feedback</Title>
-        {accessToken && !userFeedbackCard && (
+        <Title>Feedback and Rating</Title>
+        {accessToken && !userRating && (
           <div>
             <SubTitle size="1rem">
               {' '}
-              Write your invaluable feedback with{' '}
+              Rate your experience and give feedback with{' '}
               {getSkillNameFromSkillTag(skillTag)} on SUSI.AI{' '}
             </SubTitle>
-            <div>
-              <div style={{ margin: '1rem' }}>
-                <FormControl fullWidth={true}>
-                  <OutlinedTextField
-                    margin="dense"
-                    value={newFeedbackValue}
-                    label="Skill Feedback"
-                    placeholder="Skill Feedback"
-                    defaultValue=""
-                    multiline={true}
-                    fullWidth={true}
-                    onChange={this.handleFeedbackChange}
-                    aria-describedby="post-feedback-helper-text"
-                  />
-                </FormControl>
-                <Button
-                  label="Post"
-                  color="primary"
-                  variant="contained"
-                  style={{ marginTop: 10 }}
-                  handleClick={this.postFeedback}
-                  disabled={newFeedbackValue.trim().length === 0 || loading}
-                  isLoading={loading}
-                  buttonText="Post"
+            <div
```

Comment:
It can be transformed to styled component

Reply:
I think it's okay to use inline styling for one-two styling attributes.